### PR TITLE
Fix stdout noise from pkg/vtpm

### PR DESCRIPTION
**Pull request by cshari-zededa**
Signed-off-by: Hariharasubramanian C S <cshari@zededa.com>

### CODE REVIEW

File: pkg/vtpm/init.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 #Launch the VTPM server
 mkdir jail; cd jail || exit;
-/usr/bin/vtpm_server
+/usr/bin/vtpm_server > /dev/null
```
Comment (eriknordmark) [inline pkg/vtpm/init.sh:5]: I'd suggest adding a comment saying that stdout has too much noise we don't want logged.

**Review by eriknordmark — APPROVED** [review]
Let it test

File: pkg/vtpm/init.sh (hunk 1 above)
Comment (cshari-zededa) [reply pkg/vtpm/init.sh:9]: Addressed.

Comment (rvs) [reply pkg/vtpm/init.sh:9]: While I am all for this particular fix, it opens up and interesting question -- there's a few places already where we don't actually want logs to go to the cloud in a steady state, but losing them outright feels weird.

@eriknordmark -- should we open up a discussion around building some scaffolding in newlog to address the case where we just want to keep some logging on the device in a circular buffer of a certain size (for worst case local inspection) or is this not really what we should be doing anyway?

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**